system/I18n/Time.php

```diff
@@ -278,7 +278,7 @@ public static function create(int $year = null, int $month = null, int $day = nu
 	 *
 	 * @return \CodeIgniter\I18n\Time
 	 */
-	public static function createFromFormat($format, $datetime, $timeZone = null)
+	public static function createFromFormat($format, $datetime, DateTimeZone $timeZone = null)
```

Comment:
Are you actually receiving an error anywhere from this? I don't in php 7.0.22, and we want the functionality to be able to simply pass a string here.